HW18/task3/.env

```diff
@@ -0,0 +1 @@
+Image_tag="alpine_for_hw18"
```

Comment:
Do not use env variable in different registers. Upper case only

Reply:
Understood, thank you